extensions/shared/stripe-connection-notification.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { dispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import '@wordpress/notices';
+import { parse as parseUrl } from 'url';
```

Comment:
We should use `@wordpress/url` [`getQueryArg`](https://github.com/WordPress/gutenberg/blob/master/packages/url/README.md#getQueryArg).

Fine to land as-is, there are a bunch of these to migrate.